### DISCUSSION TITLE
Repository for spring-data-aerospike auto-configuration library

### DISCUSSION THREAD

**Discussion by igor-ermolenko** [aerospike/spring-data-aerospike#520] [Team Posts]
Hello @rbotzer 
We would like to create auto-configuration library for spring-data-aerospike.
Could you please create new repository for this project? 
For example with name spring-boot-data-aerospike.
Thank you!

**Comment by arrowplum** [comment]
Am about to create it. Any ideas for a better name?

**Comment by arrowplum** [comment]
https://github.com/aerospike-community/spring-boot-data-aerospike/

Let me know if you have trouble getting in or can't change the name.

**Comment by igor-ermolenko** [comment]
Thank you!
Let's leave this name. If we have better option, we will rename it in the future.
